python/ufoff.py

```diff
@@ -0,0 +1,114 @@
+
+from ufoLib2 import Font
+import flatbuffers
+from FlatFont.Ufo import FontInfo
+from enum import IntEnum
+
+SCALARS = [int, float, bool] # Other scalars needed # Enum needed
+NON_SCALARS = [str, list] # Other none-scalars needed
+
+def Capital(string: str): # seek equivalent buit-in function
+    '''Capitalizes the string without affecting other characters'''
+    return string[0].upper() + string[1:]
+
+def StartAttrVector(attr: str):
+    '''Simplifies calling buffer functions by returning a string representing the buffer function suitable for the "Getattr" function input'''
+    return "Start" + Capital(attr) + "Vector"
+
+def AddAttr(attr: str): # how about make it a subclass to str, add a method with or without @property: attr.AddAttr or attr.StartAttrVector
+    '''Simplifies calling buffer functions by returning a string representing the buffer function suitable for the "Getattr" function input'''
+    return "Add" + Capital(attr)
+
+def BuildAttrString(builder: flatbuffers.Builder, string: str):
+    '''Creates a buffer for strings'''
+    return builder.CreateString(string)
+
+def BuildAttrVector(builder: flatbuffers.Builder, attr: str ,vector: list, Value_Type: type = float):   
+    '''Creates a buffer for Vectors by passing the name of the attribute as a string'''
+    getattr(FontInfo, StartAttrVector(attr))(builder, len(vector))
+    for index in reversed(range(0, len(vector))):
```

Comment:
instead of `reversed(range(start, end))` you can also do `range(end-1, start-1, -1)` (the third parameter is the increment, which in this case actually decrements)